src/NLog/Targets/EventLogTarget.cs

```diff
@@ -244,20 +242,7 @@ protected override void InitializeTarget()
         {
             base.InitializeTarget();
 
-            var fixedSource = GetFixedSource();
-
-            if (string.IsNullOrEmpty(fixedSource))
-            {
-                InternalLogger.Debug("EventLogTarget(Name={0}): Skipping creation of event source because it contains layout renderers", Name);
-            }
-            else
-            {
-                var currentSourceName = EventLog.LogNameFromSourceName(fixedSource, MachineName);
```

Comment:
not sure if we need to remove this.

Reply:
The same as for the check on the ln:249. Also, additional checks should not be expected for the method with the suffix `IfNeeded`.